docs/admin/Troubleshooting/Operations-and-Management.md

```diff
@@ -10,6 +10,21 @@ The account details for this are found in the kubernetes secret `admin-users`. (
 
 You need to logout from CCX web-app first or clear the cookies for your `CCX_URL` in a browser.
 
+### CCX Admin API
+
+#### Basic Auth
+The following Admin API endpoints requires Basic Auth authentication method to work.
+- Billing usage → GET /admin/datastores/billing/usage/{type}
+- Users count → GET /admin/users/count
+- Datastores count → GET /admin/datastores/count
+
+The credentials can be found in the kubernetes secret `admin-users`, they are the same as for `CCX Admin UI`.
```

Comment:
it says `admin-basic-auth` not `admin-users` in the bash command. What's the correct name?